.wuf/testkits/definitions.js

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
Since wuf is common across multiple libraries - this should be documented in the wuf repo and not per file in project.

Please treat this as a public API like any open source utility that you’d use

Reply:
I agree but im not yet sure of the overall flow, so dont spend too much time on documenting.

I dont want exactly this as public api. For now its copy paste from wix-style-react but I don't think this is ideal, there should be no `definitions` thing at all. My approach is first make it work (but dont break anything), then find patterns, then APIify them (abd of course document).

So its in progress, i wont leave it like that forever :)

Reply:
So just copy-paste it to the wuf project instead of duplicating it again.